src/constants/env.ts

```diff
@@ -486,70 +495,108 @@ function parseEnv(env: Env): Config {
+      }),
     });
   }
-  if (env.NEXT_PUBLIC_PROVIDER_288) {
+  if (env.NEXT_PUBLIC_PROVIDER_V1_288) {
     providers.push({
-      url: env.NEXT_PUBLIC_PROVIDER_288,
+      source: "provider",
+      type: "Optimistic Oracle V1",
+      url: env.NEXT_PUBLIC_PROVIDER_V1_288,
       chainId: 288,
+      address: getContractAddress({
+        chainId: 288,
+        type: "Optimistic Oracle V1",
+      }),
     });
   }
-  if (env.NEXT_PUBLIC_PROVIDER_416) {
+  if (env.NEXT_PUBLIC_PROVIDER_V1_416) {
     providers.push({
-      url: env.NEXT_PUBLIC_PROVIDER_416,
+      source: "provider",
+      type: "Optimistic Oracle V1",
+      url: env.NEXT_PUBLIC_PROVIDER_V1_416,
       chainId: 416,
+      address: getContractAddress({
+        chainId: 416,
+        type: "Optimistic Oracle V1",
+      }),
     });
   }
-  if (env.NEXT_PUBLIC_PROVIDER_42161) {
+  if (env.NEXT_PUBLIC_PROVIDER_V1_42161) {
     providers.push({
-      url: env.NEXT_PUBLIC_PROVIDER_42161,
+      source: "provider",
+      type: "Optimistic Oracle V1",
+      url: env.NEXT_PUBLIC_PROVIDER_V1_42161,
       chainId: 42161,
+      address: getContractAddress({
+        chainId: 42161,
+        type: "Optimistic Oracle V1",
+      }),
     });
   }
-  if (env.NEXT_PUBLIC_PROVIDER_43114) {
+  if (env.NEXT_PUBLIC_PROVIDER_V1_43114) {
     providers.push({
-      url: env.NEXT_PUBLIC_PROVIDER_43114,
+      source: "provider",
+      type: "Optimistic Oracle V1",
+      url: env.NEXT_PUBLIC_PROVIDER_V1_43114,
       chainId: 43114,
+      address: getContractAddress({
+        chainId: 43114,
+        type: "Optimistic Oracle V1",
+      }),
     });
   }
-  if (env.NEXT_PUBLIC_PROVIDER_5) {
+  if (env.NEXT_PUBLIC_PROVIDER_V1_5) {
     providers.push({
-      url: env.NEXT_PUBLIC_PROVIDER_5,
+      source: "provider",
+      type: "Optimistic Oracle V1",
+      url: env.NEXT_PUBLIC_PROVIDER_V1_5,
       chainId: 5,
+      address: getContractAddress({ chainId: 5, type: "Optimistic Oracle V1" }),
     });
   }
-  if (env.NEXT_PUBLIC_PROVIDER_10) {
+  if (env.NEXT_PUBLIC_PROVIDER_V1_10) {
     providers.push({
-      url: env.NEXT_PUBLIC_PROVIDER_10,
+      source: "provider",
+      type: "Optimistic Oracle V1",
+      url: env.NEXT_PUBLIC_PROVIDER_V1_10,
       chainId: 10,
+      address: getContractAddress({
+        chainId: 10,
+        type: "Optimistic Oracle V1",
+      }),
     });
   }
-  if (env.NEXT_PUBLIC_PROVIDER_80001) {
+  if (env.NEXT_PUBLIC_PROVIDER_V1_80001) {
     providers.push({
-      url: env.NEXT_PUBLIC_PROVIDER_80001,
+      source: "provider",
+      type: "Optimistic Oracle V1",
+      url: env.NEXT_PUBLIC_PROVIDER_V1_80001,
       chainId: 80001,
+      address: getContractAddress({
+        chainId: 80001,
+        type: "Optimistic Oracle V1",
+      }),
     });
   }
-  // verify we have providers for all enable subgraph chains. If not checked we will have run time errors if providers are missing.
-  subgraphs.map((subgraph) => {
```

Comment:
this is no longer needed since we rely on wagmi to pull meta data and not the oracle services. each provider can now be defined to enable or disable this real time update system per chain per oracle.